### ISSUE TITLE
Missing dependencies after installing

### ISSUE THREAD

**Issue by ericcornelissen** [dkaoster/fuzzy-testing#15]
Not sure what the state of this project is, since the last commit was almost 2 years ago. However, I wanted to give it a try and found after just installing, the package is not yet usable and you get an error when trying to import it. This is because two dependencies are missing from the list of dependencies listed for the project, namely: [babel-runtime](https://www.npmjs.com/package/babel-runtime) and [prop-types](https://www.npmjs.com/package/prop-types).  Only after manually installing these, as shown below, I was able to start using this library.

```
npm install babel-runtime prop-types
```

---

As an aside, the documentation of the options is a bit unclear and only after inspecting the source code was I able to figure out what they do and what values they expect. In particular, it took me a bit to figure out that the `iterations` option needs to be increased in order for more test values to be tried (but that could be ignorance on my part). More problematic for me was the `argumentTypes` option, for which the values expected in the array are not provided, nor is it clear to me how an array with multiple values is handled.

**Comment by dkaoster** [comment]
Hi @ericcornelissen,

Sorry for the extremely late reply, and thank you for bringing this to my attention. I have recently fixed this in the latest version, and plan to incorporate more of this as feedback for future versions. Thanks!

**Comment by ericcornelissen** [comment]
Cool, can confirm it works now :+1: If you want feedback when you're working on the documentation, feel free to at-mention me :slightly_smiling_face: 

And no worries about the late reply, as I said the project didn't look very active when I posted it.

**Comment by dkaoster** [comment]
Will do!

If you don't mind me asking, I'm curious what led you to find this project? Was there a particular use case you were trying to solve? Did you end up using this or finding an alternative?

**Comment by ericcornelissen** [comment]
> If you don't mind me asking, I'm curious what led you to find this project? Was there a particular use case you were trying to solve? Did you end up using this or finding an alternative?

Not at all. Essentially, I was looking for something like [go-fuzz](https://github.com/dvyukov/go-fuzz) but for JavaScript, which I eventually found in [jsfuzz](https://gitlab.com/gitlab-org/security-products/analyzers/fuzzers/jsfuzz). That is I was looking for something that generates random inputs indefinitely with the goal of breaking the software, which this isn't quite. Anyway, along the way I looked at most NPM packages with the word "fuzz" in them, including this one. To be clear, I still think this package has merit due to the untyped nature of JavaScript.
